docs/StorageSequence.puml

```diff
@@ -0,0 +1,117 @@
+@startuml
+
+actor User
+participant ":Storage" as Storage
+participant ":File" as File
+participant ":FileWriter" as FileWriter
+participant ":Scanner" as Scanner
+participant ":ChessTile" as ChessTile
+participant ":String" as String
+
+
+create Storage
+User -> Storage++ : new Storage(): String
```

Comment:
![image](https://github.com/nus-cs2113-AY2324S1/tp/assets/82555990/f0de8914-63e3-4780-8182-a9ec49844eca)
I am not sure how your code is implemented, but it seems a bit strange to me to have user calling the command to create a new Storage() object. Did you mean user will input a command which is received by a class and the class triggers the creation of storage object? If yes, what is the class that triggers the creation of storage object? Perhaps in your uml you should have an arrow pointing from the class to storage class instead?